tests/base.py

```diff
@@ -384,3 +390,16 @@ def func_wrapper(self):
         user.roles.append(user_role)
         db_session().commit()
     return func_wrapper
+
+
+def change_admin_user_role_to_super_admin(func):
+    def func_wrapper(self):
+        user_role = Role(role='Super_Admin')
```

Comment:
This role is now `Super Admin` not `Super_Admin`

Reply:
Done @joshuaocero